src/webgl/loading.js

```diff
@@ -150,16 +152,38 @@ p5.prototype.loadModel = function(path) {
   } else if (fileType.match(/\.obj$/i)) {
     this.loadStrings(
       path,
-      strings => {
-        parseObj(model, strings);
-
-        if (normalize) {
-          model.normalize();
+      async lines => {
+        const mtlLine = lines.find(line => line.startsWith('mtllib '));
```

Comment:
I think the obj spec supports multiple mtls, so we might need to `filter` to get all the lines that have `mtllib` and then parse all of them (maybe waiting for them all via `Promise.all(...)`?)